web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -400,4 +400,55 @@ describe("profile - shared", () => {
       ).not.toBeInTheDocument();
     });
   });
+
+  describe("profile error alert", () => {
+    it.each(["STARTING", "FOREIGN"])(
+      "displays alert with the header of field that has an error when %s",
```

Comment:
[pebble] I think the test name should include "industry" since that's what's being tested